lib/modules/commands/cat.rb

```diff
@@ -0,0 +1,17 @@
+require 'open-uri'
+
+module DrawBot
+  module DiscordCommands
+    # Gives you a random cat!
+    module Cat
+      extend Discordrb::Commands::CommandContainer
+      command(:cat,
+              description: 'Gives you a random cat!',
+              usage: "#{BOT.prefix}cat") do |event|
+                catlink = JSON.parse(open('http://random.cat/meow').read)
+          event <<  "Meow! #{catlink['file'].gsub('.jpg', '')}"
```

Comment:
I would prefer the interpolation to not do any work. If you could do the `#gsub`, etc., as part of the previous or another statement that would be better.